pillar/redash.sls

```diff
@@ -6,7 +20,34 @@ apache:
       servername: redash.open-contracting.org
       context:
         proxypass: http://localhost:9090/
+
+postgres:
+  version: 13
+  # Redash connections to Postgres are not sent over the localhost connection.
+  # Postgres access is locked down further in the Linode firewall.
```

Comment:
Does this involve more than what's currently documented at https://ocdsdeploy.readthedocs.io/en/latest/develop/update/firewall.html#linode-vps-servers ?

Reply:
The ocp14 Linode firewall has a allow line granting Postgres access to it's own IPs.
I'm not sure where it would be best to document this? perhaps on the [redash page?](https://ocdsdeploy.readthedocs.io/en/latest/deploy/redash.html#create-a-redash-server)
This is a docker quirk mainly.

Reply:
Reviewing the firewall we removed the requirement for the complex "allow self" line now we've got host.docker.local working as the database connection. So the firewall is much simpler. 
I've added this to the redash page as well.